exercises/bank.js

```diff
@@ -1,25 +1,50 @@
 
 
 class Bank{
-	constructor(){
+	constructor(bankName){
+		this.name = bankName;
+		this.accountInfo = {};
 		//takes in the name of the bank.
 		//makes an object to store all accounts into
 	}
-	makeAccount( ){
+	makeAccount(accountNumber){
+		if (this.accountInfo[accountNumber]){
+			return false
+		} else {
+			var newObject = new Account();
+			this.accountInfo[accountNumber] = newObject;
+			return newObject;
+		}
 		//makes an account
 		//takes in the account number of an account
 		//checks if the account number already exists in the account storage object in the constructor
 			//if already exists, returns false
 		//otherwise makes an new Account object, storing it into the storage object in the constructor
 		//returns the Account object that was constructed
 	}
-	checkForAccount(  ){
+	checkForAccount(accountNumber){
+		if(this.accountInfo[accountNumber]){
```

Comment:
truly you don't need to return true or false, you could have returned a check

like
```
return this.accountInfo[accountNumber] !== undefined
```
or
```
return this.accountInfo.hasOwnProperty(accountNumber)
```